### PR TITLE
AADIdentityGovernanceLifecycleWorkflow

### PR DESCRIPTION
**Pull request by PiyushDub**
#### Pull Request (PR) description
AADIdentityGovernanceLifecycleWorkflow Initial Release

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/1-Create.ps1 [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADIdentityGovernanceLifecycleWorkflow "AADIdentityGovernanceLifecycleWorkflow-Onboard pre-hire employee updated version"
+        {
+            Category             = "joiner";
+            Credential           = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/1-Create.ps1]: We should use service principal (AppId, TenantId, CertThumprint) for the example. CredsCredential isn't defined anyway.

File: Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/2-Update.ps1 [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADIdentityGovernanceLifecycleWorkflow "AADIdentityGovernanceLifecycleWorkflow-Onboard pre-hire employee updated version"
+        {
+            Category             = "joiner";
+            Credential           = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/2-Update.ps1]: We should use service principal (AppId, TenantId, CertThumprint) for the example. CredsCredential isn't defined anyway.

File: Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/3-Remove.ps1 [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADIdentityGovernanceLifecycleWorkflow "AADIdentityGovernanceLifecycleWorkflow-Onboard pre-hire employee updated version"
+        {
+            Category             = "joiner";
+            Credential           = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/3-Remove.ps1]: We should use service principal (AppId, TenantId, CertThumprint) for the example. CredsCredential isn't defined anyway.

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
Need to update the examples. Other than that it all looks good!

File: Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/1-Create.ps1 (hunk 1 above)
Comment (PiyushDub) [reply Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/1-Create.ps1]: updated. thanks.

File: Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/3-Remove.ps1 (hunk 3 above)
Comment (PiyushDub) [reply Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/3-Remove.ps1]: updated.

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**